### PR TITLE
Functional tests - label one node as worker-rt

### PR DESCRIPTION
**Pull request by yanirq**
# Description
Initialize functional tests with one worker node as a worker-rt node.

### CODE REVIEW

File: features/Makefile [hunk 1]
```diff
@@ -8,6 +8,7 @@ deps-update:
 
 .PHONY: functests
 functests:
+	./hack/test_env.sh \
```
Comment (cynepco3hahue) [inline features/Makefile]: I think you should run `source ./hack/test_env.sh`

File: features/hack/test_env.sh [hunk 2]
```diff
@@ -3,4 +3,7 @@
 # Performance
 export ISOLATED_CPUS=1
 export RESERVED_CPUS=0
-export NON_ISOLATED_CPUS=1
\ No newline at end of file
+export NON_ISOLATED_CPUS=0
+
+WORKER_NODE=$(oc get nodes --no-headers -o name -l node-role.kubernetes.io/worker="" | head -n1)
+oc label $WORKER_NODE node-role.kubernetes.io/worker-rt=""
```
Comment (fedepaol) [inline features/hack/test_env.sh]: Can you add a new line here? Otherwise looks good

**Pull request merged**